Source/PeleLMeX_BCfill.H

```diff
@@ -8,6 +8,7 @@
 
 using namespace amrex;
 
+template <typename T>
```

Comment:
I don't think templating here is strictly necessary, as we always use `ProbIBC`, but it's fine to leave it because if we do move to compiling multiple cases at once then we would need the templating.

Reply:
My thought exactly.